Packs/Base/Scripts/GetIncidentsByQuery/GetIncidentsByQuery.py

```diff
@@ -1,12 +1,14 @@
+from CommonServerPython import *
+
+import math
 import pickle
-import re
 import uuid
-from datetime import datetime, timedelta
-from dateutil import parser
 
-from CommonServerPython import *
+from dateutil import parser
 
 PREFIXES_TO_REMOVE = ['incident.']
+PAGE_SIZE = int(demisto.args().get('pageSize', 500))
```

Comment:
if we go the new iterating logic we don't need that

Reply:
we need the page size anyway - it's the size of the bulk